api/types.go

```diff
@@ -510,6 +512,10 @@ func (po *PinOptions) Equals(po2 *PinOptions) bool {
 		return false
 	}
 
+	if po.ExpireAt.Format(time.RFC3339) != po2.ExpireAt.Format(time.RFC3339) {
```

Comment:
https://golang.org/pkg/time/#Time.Equal

Reply:
This is intentional, because those two times are actually not equal.
1) different monotonic clocks
2) number of nanosecond (I am using a format that doesn't have nano second precision)

I can use a format with nsec precision and fix this.

Reply:
` those two times are actually not equal`

Then why do you  consider them equal?

Monotonic clocks are only involved when doing `Time.Now`. Other than expire-in (which we do Time.Now() for the user), the time set by the user should not mutate into something different should be directly comparable to the same time set by the user on a different instance.